app/assets/javascripts/analytics.js.erb

```diff
@@ -1,7 +1,17 @@
 // Start cookie banner (display settings controlled internally by cookies)
 var element = document.querySelector('[data-module="cookie-banner"]')
+var PRODUCT_ID_PATTERN = /\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b/
+var REFERENCE_NUMBER_PATTERN = /\d{8}-\d{6}-.{6}/
+
 new GOVUK.Modules.CookieBanner().start($(element))
 
+var stripPII = function(url) {
+  url = url.replace(REFERENCE_NUMBER_PATTERN, 'reference_number')
+  url = url.replace(PRODUCT_ID_PATTERN, 'product_id')
+
+  return url
```

Comment:
I'd recommend changing this function to return `location.protocol + '//' + location.host + location.pathname`, unless we want other query params to be sent to GA.

Reply:
@bilbof I have amended.